packages/telemetry/lib/node-telemetry.js

```diff
@@ -96,6 +99,7 @@ const setupNodeHTTPTelemetry = (opts) => {
       }),
       new HttpInstrumentation(),
       new PgInstrumentation(),
+      ...additionalInstrumentations
```

Comment:
Does this need to come before our custom instrumentations so we don't accidentally clobber ours?

Reply:
Good question. I would say no so that the users can override.